.github/workflows/build-push-private.yml

```diff
@@ -107,6 +108,11 @@ jobs:
           echo "peer-results=$PEER_RESULTS"
           echo "peer-results=$PEER_RESULTS" >> "$GITHUB_OUTPUT"
 
+          # includes handling for null/missing keys
+          NET_ZERO_TARGETS="$(jq -r '.net_zero_targets | select( . != null )' $config_file)"
```

Comment:
does this modify the data in some way that one would have to manually do if they were trying to replicate this locally?